File: Data Structures & Algorithms/is-anagram/submission-0.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        int s1=1;
        int s2=1;
        for(char c: s){s1*=c-'a'+1;}
        for(char c: t){s2*=c-'a'+1;}
        cout<<s1<<endl;
        cout<<s2<<endl;
        if(s1!=s2){return false;}
        return true;
    }
};

File: Data Structures & Algorithms/find-duplicate-integer/submission-0.py
class Solution:
    def findDuplicate(self, nums: List[int]) -> int:
        sl=nums[0]
        fa=nums[nums[0]]
        while(fa!=sl):
            sl=nums[sl]
            fa=nums[nums[fa]]
        
        # sl2=0
        sl2=0
        while(sl2!=sl):
            sl=nums[sl]
            sl2=nums[sl2]
        return sl2

        
